PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -1207,7 +1207,19 @@ INT_PTR CALLBACK FindReplaceDlg::run_dlgProc(UINT message, WPARAM wParam, LPARAM
 				return TRUE;
 
 				case IDM_SEARCH_FIND:
-					goToCenter();

```

Comment:
By the way, how is it managed by the system?
Is there a way to "find" the Find dialog now?

Reply:
I tested roughly. If Find dialog is off screen but Notepad++ is on the screen, Ctrl-F will bring back Find dialog in the main frame zone of Notepad++ with or without dialog being focused.

Reply:
Indeed, I can confirm.